compiler/rustc_builtin_macros/src/assert.rs

```diff
@@ -58,7 +58,7 @@ pub fn expand_assert<'cx>(
                 path: panic_path(),
                 args: P(DelimArgs {
                     dspan: DelimSpan::from_single(call_site_span),
-                    delim: MacDelimiter::Parenthesis,
+                    delim: token::Delimiter::Parenthesis,
```

Comment:
```suggestion
                    delim: Delimiter::Parenthesis,
```
Could you avoid 3-segment paths for enum variants and import things instead (unless there are some name conflicts)?